frontend/js/i18n/messages.json

```diff
@@ -290,5 +290,297 @@
       "default-site-html": "Custom Page",
       "default-site-redirect": "Redirect"
     }
+  },
+  "es": {
+    "str": {
+      "email-address": "Dirección de correo electrónico",
+      "username": "Nombre de usuario",
+      "password": "Contraseña",
+      "sign-in": "Registrarse",
+      "sign-out": "Cerrar sesión",
+      "try-again": "Intentar otra vez",
+      "name": "Nombre",
+      "email": "Correo electrónico",
+      "roles": "Roles",
+      "created-on": "Creado: {date}",
+      "save": "Guardar",
+      "cancel": "Cancelar",
+      "close": "Cerrar",
+      "enable": "Habilitar",
+      "disable": "Inhabilitar",
+      "sure": "Si, estoy seguro",
+      "disabled": "Inhabilitado",
+      "choose-file": "Elija el archivo",
+      "source": "Fuente",
+      "destination": "Destino",
+      "ssl": "SSL",
+      "access": "Acceso",
+      "public": "Público",
+      "edit": "Editar",
+      "delete": "Borrar",
+      "logs": "Registros",
+      "status": "Estatus",
+      "online": "En línea",
+      "offline": "Desconectado",
+      "unknown": "Desconocido",
+      "expires": "Caduca",
+      "value": "Valor",
+      "please-wait": "Por favor, espere...",
+      "all": "Todo",
+      "any": "Ninguno"
+    },
+    "login": {
+      "title": "Ingrese a su cuenta"
+    },
+    "main": {
+      "app": "Nginx Proxy Manager",
+      "version": "v{version}",
+      "welcome": "Bienvenido a Nginx Proxy Manager",
```

Comment:
Bienvenido/a

Reply:
@wuilliam321  Thank you for your review :+1: 

When you refer to "Bienvenido" to a place (even a Web page) in Spanish it is said "Bienvenidos".

- https://es.hinative.com/questions/5937386
- https://forum.wordreference.com/threads/bienvenidos-a-en-tejas.1009742/
- https://www.rae.es/dpd/bienvenida

Done with commit [ff102f4](https://github.com/NginxProxyManager/nginx-proxy-manager/pull/2013/commits/ff102f41073559bf8bd0adcefaef07dec14ed3fc).